docs/blog/newsletter-february-2018.rst

```diff
@@ -0,0 +1,128 @@
+
+.. post:: February 5, 2018
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2018
+#########################################
+
+Hi everybody! Kelly O. here – back in the saddle after getting through the whole *OMG, it's a different year now* transition. 
+I'm excited to report that we have a TON of stuff happening in the community. 
+I guess we're starting this new year off strong!
+
+First things first, the march toward our Portland conference continues at a steady pace. 
+May 6-8 is starting to feel *awfully* close, and we couldn't be more excited. 
+We're working on confirming our last few speakers. 
+You'll hear from us with the full speaker line-up in the next week or so.
+*NOTE: If you're not subscribed to the North American conference mailing list, 
+`you can sign up here <http://www.writethedocs.org/newsletter/>`_.*
+
+If you're planning on coming to the conference, but you're worried about persuading your company to cover the costs, 
+you might be interested in `our new Convince Your Manager page <http://www.writethedocs.org/conf/portland/2018/convince-your-manager/>`_. 
+We've knocked together an example email and some resources that you can use to make your case.
+
+And finally, in non-conference news, we're thrilled to announce that the **Write the Docs Job Board** will be launching next week. 
+One of the benefits of the WTD community has always been its ability to connect documentarians with employers 
+who understand the value of good docs. The job board will be a destination for making those connections. 
+Keep an eye on Slack, Twitter, the forum, your email, etc. – we'll let you know as soon as it goes live.
+
+Whew! Told you there's a lot going on! But now, let's have a look at a few of the conversations that have been happening
+in `Slackland <http://www.writethedocs.org/slack/>`_, over the new year.
+
+********************************
+It's Not Personal, It's Feedback
+********************************
+
```

Comment:
Just noticed this is the only heading in title case -- others are in sentence case

Reply:
ooh, yep, good eye – will fix